src/plugins/intel_cpu/src/transformations/snippets/x64/pass/lowered/brgemm_cpu_blocking.cpp

```diff
@@ -45,23 +45,30 @@ LinearIR::constExprIt BrgemmCPUBlocking::move_new_memory_buffer(LinearIR& linear
     return std::prev(brgemm_it);
 }
 
-size_t BrgemmCPUBlocking::get_default_n_blk([[maybe_unused]] size_t n) const {
-    return dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core) ? 64 : 24;
-}
-
 std::tuple<size_t, size_t, size_t> BrgemmCPUBlocking::get_blocking_params(
     const ov::snippets::lowered::ExpressionPtr& brgemm_expr) const {
     const auto brgemm = ov::as_type_ptr<ov::intel_cpu::BrgemmCPU>(brgemm_expr->get_node());
     assert(brgemm && "BrgemmCPU is expected!");
+    const auto& brgemm_config = brgemm->get_config();
+
+    size_t m, n, k;
+    std::tie(m, n, k) = get_brgemm_dimensions(brgemm_expr);
+
+    const auto default_n_blk =
+        dnnl::impl::cpu::x64::is_superset(brgemm_config.isa(), dnnl::impl::cpu::x64::avx512_core) ? 64 : 24;
```

Comment:
Just to clarify, why have you removed `get_default_n_blk`?

Reply:
Mybe we should remove `get_default_n_blk` from the base class at all then? I'd say that virtual method `get_blocking_params` is enough to override blocking params in inherited classes